### ISSUE TITLE
fzy-native and treesitter

### ISSUE THREAD

**Issue by abrarishere** [sontungexpt/stinvim#1]
Bro Comment out fzy-native in telescope/init or uncomment the fzy-native installation.

Also comment out treesitter install all and tell user that if he want then he can uncomment that.

**Comment by sontungexpt** [comment]
I think remove fzy-native is a good idea. But treesitter, Hmm I think we never know when we will need to use a new language, so why not install it all from scratch, it will be less troublesome even though it will take up a little more memory, but then you don't need to test any more languages

**Comment by abrarishere** [comment]
Yes, I agree but as I use everything on mobile so that was unexpected

**Comment by sontungexpt** [comment]
how about setting it to all if is not mobile

**Comment by abrarishere** [comment]
Oh You can Just mention in readme
